build.gradle

```diff
@@ -59,6 +59,7 @@ subprojects {
   targetCompatibility = 1.6
 
   repositories {
+    mavenLocal()
```

Comment:
i will remove it.  I assume that allowed the build to grab the statisticVersion = '1.1-SNAPSHOT'?